installers/WindowsInstaller.hs

```diff
@@ -223,3 +230,22 @@ main opts@Options{..}  = do
     echo "Generating NSIS installer"
     procs "C:\\Program Files (x86)\\NSIS\\makensis" ["daedalus.nsi"] mempty
     signFile opts $ unpack fullName
+
+-- | Download and extract the cardano-sl windows build.
+fetchCardanoSL :: FilePath -> IO ()
+fetchCardanoSL dst = do
+  bs <- downloadCardanoSL "../cardano-sl-src.json"
+  let opts = [Zip.OptDestination dst, Zip.OptVerbose]
+  Zip.extractFilesFromArchive opts (Zip.toArchive bs)
+
+printCardanoBuildInfo :: MonadIO io => FilePath -> io ()
+printCardanoBuildInfo dst = do
+  let buildInfo what f = do
+        let f' = decodeString (dst </> f)
+        e <- testfile f'
+        when e $ do
```

Comment:
I wonder if it's valid to have a subset of these files.. and if we should catch any such divergences early.

Just a thought to consider.

Reply:
If we decide not to, maybe we could lift a type out of this?

Reply:
It's just the logging which was previously in the awful `.bat` file.

Reply:
An incremental improvement, right?